x/upgrade/types/plan.go

```diff
@@ -54,3 +71,14 @@ func (p Plan) DueAt() string {
 	}
 	return fmt.Sprintf("height: %d", p.Height)
 }
+
+// UnpackInterfaces implements UnpackInterfacesMessage.UnpackInterfaces
+func (p Plan) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
+	// UpgradedClientState may be nil
+	if p.UpgradedClientState == nil {
```

Comment:
see above comment, this seems unnecessary. I'm a little confused why unpack doesn't succeed if this returns nil as well